meta/Meta.cpp

```diff
@@ -1685,27 +1685,45 @@ sai_status_t Meta::flushFdbEntries(
     {
         // use same logic as notification, so create notification event
 
-        sai_fdb_event_notification_data_t data = {};
+        std::vector<int32_t> types;
 
-        auto *bv_id = sai_metadata_get_attr_by_id(SAI_FDB_FLUSH_ATTR_BV_ID, attr_count, attr_list);
-        auto *bp_id = sai_metadata_get_attr_by_id(SAI_FDB_FLUSH_ATTR_BRIDGE_PORT_ID, attr_count, attr_list);
         auto *et = sai_metadata_get_attr_by_id(SAI_FDB_FLUSH_ATTR_ENTRY_TYPE, attr_count, attr_list);
 
-        sai_attribute_t list[2];
+        if (et)
```

Comment:
What is someone specifies the type but sets it to TYPE_ALL?

Reply:
already fixed in my next commit